release/gcloud/Dockerfile

```diff
@@ -0,0 +1,34 @@
+# Copyright 2019 Google LLC
```

Comment:
For consistency, the existing top-level Dockerfile should be moved to release directory as well. This can be a separate PR

Reply:
Ack. Will open that PR after this is merged to avoid conflicts with the merge/squash of this change.